static/js/TopicEditor.jsx

```diff
@@ -81,52 +90,77 @@ const TopicEditor = ({origData, onCreateSuccess, close, origWasCat}) => {
          const postCategoryData = {topics: sortedSubtopics};
          requestWithCallBack({url, data: postCategoryData, setSavingStatus, redirect: () => window.location.href = "/topics"});
     }
-    const saveTopic = function () {
-        toggle();
-        let url = "";
+    const authorItems = (type) => {
+        switch(type) {
+            case 'labels':
+                return Sefaria._siteSettings.TORAH_SPECIFIC ?
+                            ["English Alternate Titles", "Hebrew Alternate Titles", "Birth Place", "Hebrew Birth Place", "Birth Year", "Place of Death", "Hebrew Place of Death", "Death Year", "Era"]
```

Comment:
Please put the lists in names variables (its very hard to understand why the different lists right now)